package io.capstone.ludendorff.components.exceptions

class DocumentExistsException: Exception("The document already exists")